pom.xml

```diff
@@ -151,6 +151,9 @@
     <!-- The port Fester listens on -->
     <fester.http.port>8888</fester.http.port>
 
+    <!-- String that should be present in A/V access URLs -->
+    <fester.av.string>pairtree</fester.av.string>
```

Comment:
While technically this works fine, I think it would be more illustrative of the type of value someone else would put in here to use the base URL for our Wowza server instead of just "pairtree", which probably won't hold much significance to someone else (other than us) reading the code.